### PR TITLE
Access control and some minor fixes

### PR DESCRIPTION
**Pull request by nitraiolo**
Added access control
Fixed config id management
Reduced logging footprint using sprintf()
Experimenting double buffering to reduce logging line messing in working memory

### CODE REVIEW

File: Firmware/ChameleonMini/Terminal/Commands.h [hunk 1]
```diff
@@ -63,9 +63,9 @@
 
 // Firmware version details.
 #define FIRMWARE_VERSION_MAJOR          1
-#define FIRMWARE_VERSION_MINOR          3
+#define FIRMWARE_VERSION_MINOR          4
 #define FIRMWARE_NAME                   "rebooted"
-#define FIRMWARE_FORK_AUTHOR            "Iceman"
+#define FIRMWARE_FORK_AUTHOR            "Nitro"
```
Comment (iceman1001) [inline Firmware/ChameleonMini/Terminal/Commands.h:68]: don't.

Comment (nitraiolo) [reply Firmware/ChameleonMini/Terminal/Commands.h:68]: Sorry, forgot to  remove this patch...
I've put this on my fork so I can see which version is running on my chameleon

**Comment by iceman1001** [comment]
Interesting,   I suggest you divide up the PR to smaller PR with dedicated changes instead of a large one with many changes.
It is easy to follow PR if they are smaller and only touches one part.  

Don't rename the firmware and try to PR it.

**Comment by nitraiolo** [comment]
OK

**Pull request closed without merging**